bin/depcheck

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
 
-/* eslint-disable no-console, prefer-arrow-callback */
+require('please-upgrade-node')(require('../package.json'));
+
+/* eslint-disable no-console, prefer-arrow-callback, comma-dangle */
```

Comment:
Why disable `comma-dangle` rule here?

Reply:
Otherwise, have to change the function call:
```diff
 require('../dist/cli')(
   process.argv.slice(2),
@@ -10,5 +12,5 @@ require('../dist/cli')(
   console.error,
   function exit(code) {
     process.exitCode = code;
-  }
+  },
 );
```

But dangling comma in function is not supported in node < 8:
```
λ node --version
v4.2.0

λ node .\bin\depcheck
E:\Projects\repos\depcheck\bin\depcheck:16
);
^

SyntaxError: Unexpected token )
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:414:25)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:134:18)
    at node.js:961:3

λ nvm use 6.11.3
Now using node v6.11.3 (64-bit)

λ node --version
v6.11.3

λ node .\bin\depcheck
E:\Projects\repos\depcheck\bin\depcheck:16
);
^

SyntaxError: Unexpected token )
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
```

Since this file isn't transpiled, we should remove the dangling comma and disable the rule.